src/components/UserActions/UserActions.ts

```diff
@@ -143,7 +143,9 @@ export class UserActions extends Component {
             (get(this.root, UserProfileModel) as UserProfileModel)
                 .getActions(this.options.userId)
                 .then(actions => (actions.length > 0 ? this.render() : this.renderNoActions()))
-                .catch(() => this.renderNoActions());
+                .catch(e => {
+                    e.statusCode === 404 ? this.renderEnablePrompt() : this.renderNoActions();
+                });
```

Comment:
Do you think you can simplify this ?

like this maybe ?
```javascript
.catch(e => (e.statusCode === 404 ? this.renderEnablePrompt() : this.renderNoActions()));
```

Reply:
That's what i had :(

Reply:
Linter says no

Reply:
👍 if this is really a pain (I, personally, don't think so), changes the rules of prettier, in another PR.